static/sass/_pattern_application.scss

```diff
@@ -0,0 +1,96 @@
+@mixin canonical-p-stepped-list {
+  .p-stepped-list--detailed {
+    counter-reset: process-list;
+    @supports (display: grid) {
+      .p-stepped-list__title {
+        grid-column-end: span 12;
+      }
+      .p-process-stepped-list__title {
+        grid-column-end: span 6;
+      };
+    }
+  }
+  
+  .p-process-stepped-list__title {
+    @extend .p-stepped-list__title;
+    
+    &::before {
+      display: none;
+    }
+  }
+
+  h3.p-process-stepped-list__title{
+    padding-left: 0;
+  }
+
+  h3 [class*="p-icon"]{
+    background-size: cover;
+    height: 2.7rem;
+    margin-right: 1rem;
+    width: 2.7rem;
+  }
+
+  .p-on-pass {
+    &::before {
+      content: counter(process-list);
+      counter-increment: process-list;
+      direction: rtl;
+      left: 0;
+      position: absolute;
+      text-align: center;
+      visibility: hidden;
+    }
+  }
+
+  .p-on-stage {
+    padding-left: 4rem;
+    &::before {
+      align-self: start;
+      background-color: #000;
+      border-radius: 100%;
+      color: #fff;
+      content: counter(process-list);
+      counter-increment: process-list;
+      direction: rtl;
+      display: block;
+      height: 2.7rem;
+      left: 0;
+      position: absolute;
+      text-align: center;
+      width: 2.7rem;
+    }
+  }
+  .p-not-on-stage {
+    padding-left: 4rem;
+    &::before {
+      align-self: start;
+      background-color: #fff;
+      border: 1px solid #000;
+      border-radius: 100%;
+      content: counter(process-list);
+      counter-increment:  process-list;
+      direction: rtl;
+      display: block;
+      height: 2.7rem;
+      left: 0;
+      position: absolute;
+      text-align: center;
+      width: 2.7rem;
+    }
+  }
+  .show-more-assessment, .show-more-early-stage, .show-more-late-stage{
+    color: blue; 
+    cursor: pointer;
+  }
+  .interview-card-done {
+    position: relative;
+    [class*="p-icon"] {
+      background-size: cover;
+      height: 1.5rem;
+      position: absolute; 
+      right: 0;
+      top: 0;
+      width: 1.5rem;
+    }
```

Comment:
Instead of custom styling you could use:
```html
<div class="p-card">
  <div class="u-float-right"><i class="p-icon--success u-align--right">Passed</i></div>
  <h4 class="p-card__title u-no-margin--bottom">Meet &amp; Greet Interview</h4>
  <p class="p-card__content">20 July, with Andreu Oller, Min Kim</p>
</div>
```